bodhi/tests/server/services/test_updates.py

```diff
@@ -768,16 +814,14 @@ def test_edit_koji_error(self):
             u'status': u'error',
             u'errors': [
                 {u'description': u'Invalid koji build: bodhi-2.0-1.fc17', u'location': u'body',
-                 u'name': u'builds'},
-                {u'description': (u'Cannot find release associated with build: bodhi-2.0-1.fc17, '
-                                  u'tags: []'),
-                 u'location': u'body', u'name': u'builds'}]}
+                 u'name': u'builds'}]}
         self.assertEqual(res.json, expected_json)
         listTags.assert_called_once_with(update.title)
 
     @unittest.skipIf(six.PY3, 'Not working with Python 3 yet')
     def test_edit_untagged_build(self):
         """Editing an update that references untagged builds should raise an error."""
+        self.maxDiff = None
```

Comment:
Let's drop this line.

Reply:
Those I quite handy, they will give you a much more detailed output when the test fails

I tend to set them for the entire class rather than per method personally